src/AXP192.cpp

```diff
@@ -5,35 +5,42 @@ AXP192::AXP192()
   
 }
 
-void AXP192::begin(void)
+void AXP192::begin(bool disableLDO2, bool disableLDO3, bool disableRTC, bool disableDCDC1, bool disableDCDC3)
 {  
     Wire1.begin(21, 22);
     Wire1.setClock(400000);
 
     // Set LDO2 & LDO3(TFT_LED & TFT) 3.0V
     Write1Byte(0x28, 0xcc);	
 
+    // Set ADC sample rate to 200hz
```

Comment:
Why increase the sample rate to 200Hz? Just curious about the need.
This "might" increase AXP192 idle current consumption. 
Although, the measurement for bat Coulomb might be more accurate?